### ISSUE TITLE
Missing spaces after some punctuation marks in German UI

### ISSUE THREAD

**Issue by 7aqdxe6f** [borgbase/vorta#974]
**Describe the bug**

The spaces that should be visible after some punctuation marks are missing in the German UI. This only seems to affect `.` and `,`, not: `:` or `)`.

**Environment:**
- OS: macOS 11.3.1
- Vorta version: 0.7.5
- Installed from: brew

**Comment by m3nu** [comment]
Any screenshots of examples?

**Comment by 7aqdxe6f** [comment]
Hi, thank you for your fast reply!

I did some further testing: Checked a few past releases on my Mac and tried installing newest releases both from .dmg and brew, issue persists. Also persists across multiple languages.

I therefore think this is most likely caused by some program/setting/config on my Mac and not due to an issue with the app after all.

Therefore it might be best to just wait a bit and close the issue if others aren't reporting similar problems.

PS: Can not post a screenshot, sorry. Don't have the time to look into the implications of the project license for posting screenshots. Hope you understand.

I really hope I'm not causing any inconvenience.
